src/Commands/MachineToken/DeleteAllCommand.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\MachineToken;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+
+class DeleteAllCommand extends TerminusCommand
+{
+
+  /**
+   * Delete all stored machine tokens and log out.
+   *
+   * @command machine-token:delete-all
+   * @aliases mt:delete-all
+   **
+   * @usage terminus machine-token:delete-all
+   *   Deletes all of the stored machine tokens the current user out.
+   */
+    public function deleteAll()
```

Comment:
Yeah. Probably overstepping the bounds of the task, but a proper `clear:cache` is essential with the command cache. Can't have people deleting all of their machine tokens when they install a new plugin